### PR TITLE
Clean archived and redirected links

### PR DESCRIPTION
**Pull request by Ly0n**
* Cleaning archived projects here: https://gist.github.com/andrew/0ab4d626dd98e7843d991f44f497273e
* Cleaning redirected / dead links based on [awesome_bot](https://github.com/dkhamsing/awesome_bot)
* Removed awesome_bot CI since this this needs to be done automated to correct all links.
* Remove toot CI job

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ If you're looking for developer tutorials on topics such as renewable energy, cl
 - [pvtrace](https://github.com/danieljfarrell/pvtrace) - Optical ray tracing for luminescent materials and spectral converter photovoltaic devices.
 - [SolarPILOT](https://github.com/NREL/SolarPILOT) - Solar power tower layout and optimization tool.
 - [solar-data-tools](https://github.com/slacgismo/solar-data-tools) - Data analysis tools for working with historical PV solar time-series data sets.
-- [SolarPV-DER-simulation-utility](https://github.com/tdcosim/SolarPV-DER-simulation-utility) - Allows user to run dynamics simulations for solar photovoltaic distributed energy resource connected to a stiff voltage source or to an external program.
+- [SolarPV-DER-simulation-utility](https://github.com/tdcosim/SolarPV-DER-simulation-tool) - Allows user to run dynamics simulations for solar photovoltaic distributed energy resource connected to a stiff voltage source or to an external program.
```
Comment (andrew) [inline README.md:114]: Does the name need updating here as well?

File: README.md [hunk 2]
```diff
@@ -181,7 +179,7 @@ If you're looking for developer tutorials on topics such as renewable energy, cl
 - [windfarmGA](https://github.com/YsoSirius/windfarmGA) - Genetic algorithm to optimize the layout of wind farms.
 - [wtphm](https://github.com/lkev/wtphm) - The Wind Turbine Prognostics and Health Management library processes wind turbine events data, as well as operational SCADA data for easier fault detection, prognostics or reliability research.
 - [AirfoilPreppy](https://github.com/WISDEM/AirfoilPreppy) - A Python module for pre-processing and evaluating aerodynamic airfoil data, primarily for wind turbine applications.
-- [GreenGuard](https://github.com/signals-dev/GreenGuard) - A collection of end-to-end solutions for machine learning problems commonly found in monitoring wind energy production system.
+- [GreenGuard](https://github.com/sintel-dev/Draco) - A collection of end-to-end solutions for machine learning problems commonly found in monitoring wind energy production system.
```
Comment (andrew) [inline README.md:182]: Does the name need updating here as well?

File: README.md [hunk 3]
```diff
@@ -294,7 +292,7 @@ If you're looking for developer tutorials on topics such as renewable energy, cl
 - [openTEPES](https://github.com/IIT-EnergySystemModels/openTEPES) - Determines the investment plans of new facilities (generators, ESS and lines) for supplying the forecasted demand at minimum cost.
 - [PowerGenome](https://github.com/PowerGenome/PowerGenome) - A tool to quickly and easily create inputs for power systems models.
 - [load_forecasting](https://github.com/pyaf/load_forecasting) - Load forecasting on Delhi area electric power load using ARIMA, RNN, LSTM and GRU models.
-- [REopt_Lite_API](https://github.com/NREL/REopt_Lite_API) - Offers a subset of features from NREL's more comprehensive REopt model. Both models provide concurrent, multiple technology integration and optimization capabilities to help organizations meet their cost savings and energy performance goals.
+- [REopt_Lite_API](https://github.com/NREL/REopt_API) - Offers a subset of features from NREL's more comprehensive REopt model. Both models provide concurrent, multiple technology integration and optimization capabilities to help organizations meet their cost savings and energy performance goals.
```
Comment (andrew) [inline README.md:295]: Does the name need updating here as well?

File: README.md [hunk 4]
```diff
@@ -823,11 +804,11 @@ energy system designs and analysis of interactions between technologies.
 - [CDRMEx](https://github.com/hsbay/cdrmex) - Carbon Dioxide Removal Modeling Experiments.
 - [OpenIAM](https://gitlab.com/NRAP/OpenIAM) - An open source integrated assessment model developed by National Risk Assessment Partnership Phase II to facilitate risk assessment, management and containment assurance for geologic carbon sequestration projects.
 - [FOQUS](https://github.com/CCSI-Toolset/FOQUS) - Framework for Optimization and Quantification of Uncertainty and Surrogates.<!--lint ignore awesome-spell-check-->
-- [GEOSX](https://github.com/GEOSX/GEOSX) - A simulation framework for modeling coupled flow, transport, and geomechanics in the subsurface.
+- [GEOSX](https://github.com/GEOS-DEV/GEOS) - A simulation framework for modeling coupled flow, transport, and geomechanics in the subsurface.
```
Comment (andrew) [inline README.md:807]: Does the name need updating here as well?

File: README.md [hunk 5]
```diff
@@ -847,18 +828,17 @@ energy system designs and analysis of interactions between technologies.
 - [Methane-detection-from-hyperspectral-imagery](https://github.com/satish1901/Methane-detection-from-hyperspectral-imagery) - Deep Learning based Remote Sensing Methods for Methane Detection in Airborne Hyperspectral Imagery.
 - [Methane Source Finder](https://methane.jpl.nasa.gov/) - Explore, analyze, and download methane plumes detected from airborne platforms on an interactive map alongside VISTA infrastructure, gridded methane estimates, and other additional data layers.
 - [deep-smoke-machine](https://github.com/CMU-CREATE-Lab/deep-smoke-machine) - Deep learning models and dataset for recognizing industrial smoke emissions.
-- [Ribbit Network Frog Sensor](https://github.com/Ribbit-Network/ribbit-network-frog-sensor) - The sensor for the world's largest crowdsourced network of open-source, low-cost, CO2 Gas Detection Sensors.
+- [Ribbit Network Frog Sensor](https://github.com/Ribbit-Network/ribbit-network-frog-hardware) - The sensor for the world's largest crowdsourced network of open-source, low-cost, CO2 Gas Detection Sensors.
```
Comment (andrew) [inline README.md:831]: Does the name need updating here as well?

File: README.md [hunk 6]
```diff
@@ -921,7 +901,7 @@ energy system designs and analysis of interactions between technologies.
 - [pycirk](https://github.com/CMLPlatform/pycirk) - Model Circular Economy policy and technological interventions in Environmentally Extended Input-Output Analysis.
 - [Global Plastic Navigator](https://github.com/WWF-Deutschland/marine-plastic-explorer) - Visualises the most recent and high-resolution data of current scientific publications on marine plastic pollution.
 - [marine_debris_ML](https://github.com/NASA-IMPACT/marine_debris_ML) - Marine debris detection with commercial satellite imagery and deep learning.
-- [ADVECTOR](https://github.com/TheOceanCleanupAlgorithms/ADVECTOR) - A whole-ocean marine debris transport model which is built to handle millions of particles and terabytes of data.
+- [ADVECTOR](https://github.com/TheOceanCleanupAlgorithms/ADVECT) - A whole-ocean marine debris transport model which is built to handle millions of particles and terabytes of data.
```
Comment (andrew) [inline README.md:904]: Does the name need updating here as well?

File: README.md [hunk 7]
```diff
@@ -1466,7 +1444,7 @@ energy system designs and analysis of interactions between technologies.
 ### Radiative Transfer
 
 - [lowtran](https://github.com/space-physics/lowtran) - Atmospheric absorption extinction, scatter and irradiance model in Python and Matlab.
-- [RadiativeTransfer.jl](https://github.com/RadiativeTransfer/RadiativeTransfer.jl) - A full end-to-end modular software suite for radiative transfer and related atmospheric analysis.
+- [RadiativeTransfer.jl](https://github.com/RemoteSensingTools/vSmartMOM.jl) - A full end-to-end modular software suite for radiative transfer and related atmospheric analysis.
```
Comment (andrew) [inline README.md:1447]: Does the name need updating here as well?

File: README.md [hunk 8]
```diff
@@ -1752,7 +1727,7 @@ energy system designs and analysis of interactions between technologies.
 - [openSenseMap](https://github.com/sensebox/openSenseMap) - Wikipedia for environmental data in order to awaken and promote education, environmental and climate protection.
 - [rfasst](https://github.com/JGCRI/rfasst) - Estimation of a consistent range of adverse health and agricultural effects attributable to air pollution for a GCAM scenario.
 - [saqgetr](https://github.com/skgrange/saqgetr) - Import Air Quality Monitoring Data in a Fast and Easy Way.
-- [Ozone](https://github.com/Milind220/Ozone) - An open-source package to easily obtain real-time, historical, or forecasted air quality data for anywhere in the world.
+- [Ozone](https://github.com/Ozon3Org/Ozon3) - An open-source package to easily obtain real-time, historical, or forecasted air quality data for anywhere in the world.
```
Comment (andrew) [inline README.md:1730]: Does the name need updating here as well?

File: README.md [hunk 9]
```diff
@@ -1762,20 +1737,19 @@ energy system designs and analysis of interactions between technologies.
...
+- [nhdplusTools](https://github.com/DOI-USGS/nhdplusTools) - This package is a growing set of tools for manipulation of hydrographic data using the NHDPlus data model.
 - [The Safe Water Project](https://github.com/codeforboston/safe-water) - A team of volunteers at Code for Boston who are using data modeling, data visualization and machine learning to predict, visualize, and share data about the presence of hazardous drinking and surface water contaminants in the United States.
 - [HydroFunctions](https://github.com/mroberge/hydrofunctions) - A suite of convenience functions for working with hydrology data in an interactive Python session.
-- [CWATM](https://github.com/CWatM/CWatM) - Represents one of the new key elements of IIASAs Water program to assess water supply, water demand and environmental needs at global and regional level.
+- [CWATM](https://github.com/PeterBurek/oldCwat) - Represents one of the new key elements of IIASAs Water program to assess water supply, water demand and environmental needs at global and regional level.
```
Comment (andrew) [inline README.md:1748]: Should be updated to https://github.com/iiasa/CWatM instead?

File: README.md [hunk 10]
```diff
@@ -1858,7 +1832,7 @@ energy system designs and analysis of interactions between technologies.
 - [US_SoilGrids100m](https://github.com/aramcharan/US_SoilGrids100m) - Complete-coverage gridded predictions of soil properties (percent organic carbon, total nitrogen, bulk density, pH, and percent sand and clay) and classes (taxonomic great group and particle size in the control section) for the Conterminous U.S.
 - [pytesmo](https://github.com/TUW-GEO/pytesmo) - Python Toolbox for the Evaluation of Soil Moisture Observations.
 - [Drought Indices Map](https://github.com/imartinezl/drought-map) - This project attempts to find an accessible and friendly way to visualize data of drought indices in Spain from 1961 until 2017.
-- [ml_drought](https://github.com/esowc/ml_drought) - A Machine Learning Pipeline to Predict Vegetation Health.
+- [ml_drought](https://github.com/ECMWFCode4Earth/ml_drough) - A Machine Learning Pipeline to Predict Vegetation Health.
```
Comment (andrew) [inline README.md:1835]: typo, missing a `t` on the end

File: README.md [hunk 11]
```diff
@@ -1964,10 +1937,9 @@ energy system designs and analysis of interactions between technologies.
...
 - [Stripe Climate Carbon Removal Purchases](https://github.com/stripe/carbon-removal-source-materials) - Source materials supporting Stripe Climate carbon removal purchases.
-- [Salish Sea Wiki](https://salishsearestoration.org/) - A wiki oriented around human systems and ecological restoration (usually riparian / watershed) for the Salish Sea area (Northwestern US & BC Canada).
 - [Justice40 Tool](https://github.com/usds/justice40-tool) - A tool to identify disadvantaged communities due to environmental, socioeconomic and health burdens.
 - [Net Zero Tracker](https://zerotracker.net/) - Create the definitive global resource for collating, assessing and presenting the scale and quality of net zero pledges across nationals, sub-nationals, companies and other entities.
-- [Credible Goals Database](https://www.embeddingproject.org/goals-database) - A public goals database containing leading sustainability goals and commitments set by large companies globally.
+- [Credible Goals Database](http://embeddingproject.org/goals-database) - A public goals database containing leading sustainability goals and commitments set by large companies globally.
```
Comment (andrew) [inline README.md:1942]: should still be https

**Review by andrew — CHANGES_REQUESTED** [review]
A few tweaks needed

File: README.md (hunk 1 above)
Comment (Ly0n) [reply README.md:114]: Eagle eye :eagle:

**Comment by Ly0n** [comment]
I completely overlooked the fact that a project can also have a new name with a new URL. Thank you for being so thoughtful!

**Review by andrew — APPROVED** [review]

**Pull request merged**